x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -107,23 +108,26 @@ const RuleEdit = lazy(() => import('../../rule_form/rule_edit'));
 
 export interface RulesListProps {
```

Comment:
are we trying to do an alphabetical order? If yes, it looks like we still have some wrong here ;)